observatory-platform/observatory/platform/docker/entrypoint-airflow.sh.jinja2

```diff
@@ -11,21 +27,38 @@ if [ $1="webserver" ]; then
   airflow create_user -r Admin -u airflow -e ${AIRFLOW_UI_USER_EMAIL} -f Observatory -l Admin -p ${AIRFLOW_UI_USER_PASSWORD}
 fi
 
+###############################
+# Install observatory-platform
+###############################
 # Enter observatory platform folder
 cd /opt/observatory/observatory-platform
 
-# Install Python dependencies for Observatory Platform
-pip3 install -r requirements.txt --user
-
+# TODO: remove this
 # Remove tests directory which other packages install to sometimes
 TESTS_DIRECTORY='/home/airflow/.local/lib/python3.7/site-packages/tests'
 if [ -d ${TESTS_DIRECTORY} ]; then
   rm -r ${TESTS_DIRECTORY}
 fi
```

Comment:
Ah I must have forgotten about that. Once I've finished the changes for the DEBUG_FILE_PATH we should be able to remove it because we won't need to depend on the tests package anymore.